wpilibjIntegrationTests/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     compile 'com.googlecode.junit-toolbox:junit-toolbox:2.0'
     compile 'org.apache.ant:ant:1.9.4'
     compile 'org.apache.ant:ant-junit:1.9.4'
+    compile files(wpilibj.sourceSets.test.output.classesDir)
 }
 
+compileJava.dependsOn tasks.getByPath(':wpilibj:testClasses')
```

Comment:
This makes it so that the test classes get compiled first.
Basically, its saying that this task is dependent. The line above is indicating what file from that task we want.